lib/metasploit/cache/version.rb

```diff
@@ -9,9 +9,11 @@ module Version
       # The major version number.
       MAJOR = 0
       # The minor version number, scoped to the {MAJOR} version number.
-      MINOR = 77
+      MINOR = 79
       # The patch version number, scoped to the {MAJOR} and {MINOR} version numbers.
-      PATCH = 0
+      PATCH = 1
+      # The prerelease version, scoped to the {MAJOR}, {MINOR}, and {PATCH} version numbers.
+      PRERELEASE = 'load-by-reference-name-payload-staged-class'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.